include/tvm/ir/op.h

```diff
@@ -244,12 +244,18 @@ class OpRegEntry {
       runtime::TypedPackedFunc<bool(const Array<Type>&, int, const Attrs&, const TypeReporter&)>
           type_rel_func);
   /*!
-   * \brief Set the the attrs type key and index to be AttrsType.
+   * \brief Set the attrs type key and index to be AttrsType.
    * \tparam AttrsType the attribute type to b set.
    * \return reference to self.
    */
   template <typename AttrsType>
   inline OpRegEntry& set_attrs_type();
+  /*!
+   * \brief Set the attrs type key and index to be AttrsType.
+   * \tparam key The attribute type key to be set.
```

Comment:
I think this should be `\param` rather than `\tparam`?

Reply:
Thanks

Reply:
> I think this should be `\param` rather than `\tparam`?

\tparam means template param, so tparam is ok